src/FilteredSearch.tsx

```diff
@@ -18,7 +18,6 @@ const FilteredSearch = styled.div<SxProp>`
     border-radius: 0;
     border-top-right-radius: ${get('radii.2')};
     border-bottom-right-radius: ${get('radii.2')};
-    z-index: 1; // Allows the focus outline to show on top of the dropdown.
```

Comment:
Now [ the Dropdown is deprecated](https://primer.style/react/deprecated/Dropdown#deprecation) and z-index causes an issue as I mentioned on the description, this z-index might not be necessary?